api-admin/src/test/java/bio/terra/pearl/api/admin/service/notifications/NotificationConfigExtServiceTests.java

```diff
@@ -62,4 +75,184 @@ public void testNotificationConfigReplace(TestInfo testInfo) {
         notificationConfigService.find(oldConfig.getId()).orElseThrow();
     assertThat(updatedOldConfig.isActive(), equalTo(false));
   }
+
+  @Test
+  @Transactional
```

Comment:
this and the following test will be less necessary if you delegate to the intra-service 'find' method above